src/zalopay/__tests__/__mocks__/disbursement/responses.ts

```diff
@@ -0,0 +1,64 @@
+export const queryMerchantBalanceSuccess = {
+    "return_code": 1,
+    "return_message": "Giao dịch thành công.",
```

Comment:
@longxice should all messages be written in English?

Reply:
@anhldbk sure a